### PR TITLE
Add bg/character sprite selector with preview

### PR DESCRIPTION
**Pull request by WiIIiam278**
image

### CODE REVIEW

**Comment by WiIIiam278** [comment]
uhh whoops

**Review by jonko0493 — APPROVED** [review]

**Comment by jonko0493** [comment]
This looks amazing! Fantastic work!

File: src/SerialLoops/Controls/ItemListPanel.cs [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 using Eto.Forms;
 using HaruhiChokuretsuLib.Util;
-using SerialLoops.Lib;
 using SerialLoops.Lib.Items;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using SerialLoops.Utility;
```
[the remark this answers is not in the document]
Comment (WiIIiam278) [reply src/SerialLoops/Controls/ItemListPanel.cs:8]: yeah lol whoops, this is me doing dev on Rider which has totally different C# style rules

File: src/SerialLoops/Utility/ControlGenerator.cs [hunk 2]
```diff
@@ -28,5 +31,34 @@ public static TableLayout GetControlWithLabelTable(string title, Control control
...
+            return new StackLayout
+            {
+                Orientation = Orientation.Horizontal,
+                HorizontalContentAlignment = HorizontalAlignment.Left,
+                VerticalContentAlignment = VerticalAlignment.Center,
+                Spacing = 5,
+                Items =
+                {
+                    new ImageView { Image = new Bitmap(GetItemIcon(type, log)) },
+                    fileName
```
[the remark this answers is not in the document]
Comment (WiIIiam278) [reply src/SerialLoops/Utility/ControlGenerator.cs]: image
nifty!

**Pull request merged**